zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -128,6 +128,9 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
             else:
                 example = json.dumps(argument["content"]["application/json"]["example"])
 
+            if str(example) == "True" or str(example) == "False":
+                example = str(example).lower()
+
```

Comment:
`json.dumps` on `example` without content-type `application/json` adds quotation marks to words. so this is the simplest solution I found for this.